lib/modules/manager/github-actions/extract.ts

```diff
@@ -193,6 +198,89 @@ function extractRunners(runner: unknown): PackageDependency[] {
   return runners.map(extractRunner).filter(isNotNullOrUndefined);
 }
 
+const communityActions = [
+  {
+    // https://github.com/jaxxstorm/action-install-gh-release
+    use: /^jaxxstorm\/action-install-gh-release@.*$/,
+    schema: z
+      .object({ with: z.object({ repo: z.string(), tag: z.string() }) })
+      .transform(({ with: val }): PackageDependency => {
+        return {
+          datasource: GithubReleasesDatasource.id,
+          depName: val.repo,
+          packageName: val.repo,
+          currentValue: val.tag,
+          depType: 'uses-with',
+        };
+      }),
+  },
+  {
+    // https://github.com/astral-sh/setup-uv
+    use: /^astral-sh\/setup-uv@.*$/,
```

Comment:
this manager supports gitea / forgejo actions too. there this could be the full repo url.